mitiq/pt/pt.py

```diff
@@ -3,28 +3,64 @@
 # This source code is licensed under the GPL license (v3) found in the
 # LICENSE file in the root directory of this source tree.
 
-from typing import Callable, List, Optional, Tuple, Union
+from typing import Callable, List, Optional, Tuple, Union, cast
 
+import random
 import cirq
 import numpy as np
 
 from mitiq import QPROGRAM, Executor, Observable, QuantumResult
 from mitiq.interface import atomic_converter
 
+Paulis = Union[cirq.IdentityGate, cirq.XPowGate, cirq.YPowGate, cirq.ZPowGate]
```

Comment:
Looks like this type isn't being used anymore. I tried to be specific when defining this type, but perhaps it's not needed as `cirq.Gate` covers all the Pauli's (and more). What I've seen about type-hinting is "accept general, return specific" meaning be as general as you can type-hinting function arguments (that users may use), and as specific as possible type-hinting return types. WDYT?

Reply:
I found Paulis wasn't functioning as a type, so we didn't get any inheritance or annotation. I think if we want to define a specific return type we'll want to build it in cirq with [`GateFamily`](https://quantumai.google/reference/python/cirq/GateFamily)